packages/@tinacms/cli/src/next/config-manager.ts

```diff
@@ -166,6 +168,14 @@ export class ConfigManager {
       this.generatedFolderPath,
       'client.js'
     )
+    this.generatedDatabaseClientTSFilePath = path.join(
+      this.generatedFolderPath,
+      'backendClient.ts'
```

Comment:
my vote would be to call this `databaseClient` as it is more specific than 'backend' - and also whenever you import it, the import ends up being called databaseClient

Reply:
Yeah I like `databaseClient`  better as well.